geode-core/src/test/resources/org/apache/geode/management/internal/cli/commands/golden-help-offline.properties

```diff
@@ -2719,14 +2719,14 @@ PARAMETERS\n\
 \ \ \ \ \ \ \ \ Sets the IP address the Geode memcached service listens on for memcached clients. The\n\
 \ \ \ \ \ \ \ \ default is to bind to the first non-loopback address for this machine.\n\
 \ \ \ \ \ \ \ \ Required: false\n\
-\ \ \ \ redis-port\n\
-\ \ \ \ \ \ \ \ Sets the port that the Geode Redis service listens on for Redis clients.\n\
+\ \ \ \ compatible-with-redis-port\n\
+\ \ \ \ \ \ \ \ Sets the port that the Geode API for Compatibility with Redis service listens on for Compatibility with Redis clients.\n\
 \ \ \ \ \ \ \ \ Required: false\n\
-\ \ \ \ redis-bind-address\n\
-\ \ \ \ \ \ \ \ Sets the IP address the Geode Redis service listens on for Redis clients. The default is to\n\
+\ \ \ \ compatible-with-redis-bind-address\n\
+\ \ \ \ \ \ \ \ Sets the IP address the Geode API for Compatibility with Redis service listens on for Compatibility with Redis clients. The default is to\n\
```

Comment:
Sets the IP address the Geode APIs compatible with Redis service listens on for Redis clients. The default is to